cli/Cargo.toml

```diff
@@ -120,11 +120,12 @@ winapi = { version = "=0.3.9", features = ["knownfolders", "mswsock", "objbase",
 [dev-dependencies]
 deno_bench_util = { version = "0.51.0", path = "../bench_util" }
 flaky_test = "=0.1.0"
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
We are not going to add OpenSSL dependency.

Reply:
More helpful feedback: use `rustls` to generate the RSA key. You can view some code in `ext/crypto` for how this is done there.

Reply:
I've found that `trust-dns` also have builtin function to generate RSA keys but it's on openssl lib inside.